x-pack/packages/security-solution/side_nav/src/solution_side_nav_panel.test.tsx

```diff
@@ -43,7 +43,6 @@ const mockItems: SolutionSideNavItem[] = [
     label: 'Kubernetes',
     href: '/kubernetes',
     description: 'Kubernetes description',
-    isBeta: true,
```

Comment:
I think we need to leave the Beta tag on until we can bring the Defend for Containers integration to GA. We won't be able to guarantee GA quality of the dashboard until the datasource for it is also GA.